### DISCUSSION TITLE
When OTA 2021.36, how to adjust electrical current

### DISCUSSION THREAD

**Discussion by mrgaolei** [timdorr/tesla-api#455] [Q&A]
There be 2 new API

1. detect max current is 16A or 32A
2. set current from 5A to 32A

**Comment by timdorr** [comment]
There's a new API for this: https://github.com/timdorr/tesla-api/blob/master/docs/miscellaneous/endpoints.md?plain=1#L112-L116

**Reply by salvesen** [reply]
@timdorr whats the parameter name? Current?

**Comment by salvesen** [comment]
Anybody know the parameter name? "current" is not working for me..

**Reply by GaPhi** [reply]
Did you tried "charging_amps" as logged in com.teslamotors.tesla/cache/logs/log.* files?

**Reply by salvesen** [reply]
@GaPhi thank you, that worked :D

**Reply by GaPhi** [reply]
Good, I was waiting for this feature for so long time !

I was planning its develepment for the next days but you already have all in your hands now, I let you do it then, it should be easy/quick.
